defaults/main.yml

```diff
@@ -58,3 +58,11 @@ gitlab_email_enabled: "false"
 gitlab_email_from: "gitlab@example.com"
 gitlab_email_display_name: "Gitlab"
 gitlab_email_reply_to: "gitlab@example.com"
+
+# LetsEncrypt configuration
+gitlab_letsencrypt: false
+gitlab_letsencrypt_contact_email: "gitlab@example.com"
```

Comment:
See #148 — this should be `_emails` and allow a list, and then when it's printed you could use the `| to_json` filter so it's printed in a way that works in the ruby config file.